content/20.treatments.md

```diff
@@ -86,6 +86,68 @@ A list of current studies and their results, using carefully the information req
 
 Summarize the state of the antiviral approach.
 
+##### Hydroxychloroquine and Chloroquine
+
+Hydroxychloroquine (HCQ) and chloroquine (CQ) are quinolones that have been widely used as anti-malarial and autoimmune drugs in humans.
```

Comment:
Hey Feng! Super minor but "used as anti-malarials and drugs against autoimmune disease" might be slightly more clear. "Autoimmune drugs" almost makes it sound like the drugs themselves are autoimmune.

Reply:
Could just call it a DMARD (disease-modifying anti-rheumatic drug) for autoimmune diseases like lupus.

Reply:
Thanks for the comment, Diane! I think it's incredibly important to be as clear as possible and have made the change recommended by Rishi.